tests/test_closing.py

```diff
@@ -297,6 +297,58 @@ def test_closing_negotiation_reconnect(node_factory, bitcoind):
     assert bitcoind.rpc.getmempoolinfo()['size'] == 1
 
 
+@unittest.skipIf(not DEVELOPER, "needs DEVELOPER=1")
+def test_closing_specified_destination(node_factory, bitcoind):
+    l1, l2 = node_factory.line_graph(2)
+    chan = l1.get_channel_scid(l2)
+
+    l1.pay(l2, 200000000)
+
+    bitcoind.generate_block(5)
+    addr = 'bcrt1qeyyk6sl5pr49ycpqyckvmttus5ttj25pd0zpvg'
+    l1.rpc.close(chan, destination=addr)
```

Comment:
is there a way to check ordered params in addition to key-value ones with the python rpc? this really needs a test for the single value 'destination' ie

```
close id destination
```

Reply:
Yes, you can feed it directly like so:

```python
l1.rpc.call('close', [chan, addr])
l1.rpc.call('close', {'destination': addr})
```

Reply:
I’ll add this test in the tests for compat.

Reply:
@niftynei @rustyrussell I find `l1.rpc.call('close', [chan, addr])` doesn't work. It should be `l1.rpc.call('close', [chan, None, addr])` like @rsbondi said.
And I also test the form of `fundchannel_start`:
PASSED:        l1.rpc.call("fundchannel_start", [l2.info['id'], 10**6, None, True])
FAILED:         l1.rpc.call("fundchannel_start", [l3.info['id'], 10**6, True])

The result of `CLI` is the same. (replace `None` to `null`)